src/ui/App/App.tsx

```diff
@@ -257,10 +269,10 @@ function Views({ initialRoute }: { initialRoute?: string }) {
             }
           />
           <Route
-            path="/backup-wallet/*"
+            path="/reveal-private-key/*"
             element={
               <RequireAuth>
-                <BackupWallet />
+                <RevealPrivateKey />
```

Comment:
The `RevealPrivateKey` page/component is essentially the old BackupWallet, but stripped down to only include the "reveal private key" functionality. I also changed the route to better reflect its purpose

Reply:
Thanks for clarifying